prost-build/src/lib.rs

```diff
@@ -125,6 +125,10 @@
 //! // Now compile your proto files via prost-build
 //! ```
 //!
+//! ## Feature Flags
```

Comment:
Details about some of these features are currently placed before this. Perhaps place this first and reference the respective details sections?

Reply:
Good idea, I moved the Feature Flags section